src/azure-cli-core/setup.py

```diff
@@ -49,7 +49,7 @@
     'colorama~=0.4.1',
     'humanfriendly>=4.7,<9.0',
     'jmespath',
-    'knack==0.7.2',
+    'knack==0.8.0rc1',
```

Comment:
I tried to use `~=0.8.0rc1` but `0.8.1` won't fit (it should). 

```py
>>> from packaging.specifiers import SpecifierSet
>>> "0.8.1" in SpecifierSet("~=0.8.0rc1")
False
```

This behavior contradicts [PEP 440](https://www.python.org/dev/peps/pep-0440/#compatible-release), and this bug is mentioned at https://github.com/pypa/packaging/issues/100.